### DISCUSSION TITLE
wureset

### DISCUSSION THREAD

**Discussion by FRECAUT** [community/community#65689] [Packages]
### Select Topic Area

General

### Body

bonsoir,
comment installer wurest en francais?
je vous remercie.
cordilement

**Comment by Akash1134** [comment]
Hi @FRECAUT ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project.  It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!h
